### DISCUSSION TITLE
How can I install a relase for Firefox?

### DISCUSSION THREAD

**Discussion by Starhowl** [birchill/10ten-ja-reader#607] [General]
I've downloaded [https://github.com/birtles/rikaichamp/releases/tag/v0.5.7](url) but there's no XPI-file inside the archive rikaichamp-0.5.7-firefox.zip?

Can someone please explain to me on how to make this addon run on a Windows-Firefox without resorting to the official outdated version on the official Firefox-Addon website?

**Comment by birtles** [comment]
I'm not really sure but I believe XPI files are just ZIP files with a different extension. So I believe you can just download https://github.com/birtles/rikaichamp/releases/download/v0.5.7/rikaichamp-0.5.7-firefox.zip and rename it to `rikaichamp-0.5.7.xpi`.

Also, I'm curious about what is outdated on the official Firefox website? It should show the 0.5.7 version by now (there's about a 10 minute delay between when the release is created here and when AMO is updated).

**Reply by Starhowl** [reply]
Weird, I could swear that my extension wouldn't update from 0.5.3 to 0.5.7 and the Firefox-page didn't show a newer version either?

**Reply by birtles** [reply]
Yeah, there is definitely a delay in the update to AMO. I think it only went through about one or two minutes ago.

(Also, I didn't release anything after 0.5.3 for Firefox since subsequent versions 0.5.4 and 0.5.5 only included changes to the Chromium-based browser packages.)
